Makefile

```diff
@@ -0,0 +1,33 @@
+PREFIX ?= /usr/local
+BASH_COMPLETION_DIR := $(shell pkg-config --variable=completionsdir bash-completion) 
```

Comment:
I'm sure I tested this before, on this OSX laptop, no idea why it wasn't breaking then...